packages/versioning/test/versioned-dependencies.test.ts

```diff
@@ -402,366 +402,6 @@ describe("versioning: reference versioned library", () => {
   });
 });
 
-describe("versioning (deprecated): reference versioned library (@versionedDependency)", () => {
```

Comment:
Do we have this same coverage in the useDependency tests?

Reply:
We do.